app/src/main/res/values/strings.xml

```diff
@@ -813,5 +813,6 @@
 
   <!-- Custom drugs search -->
   <string name="customdrugssearch_hint">Enter medicine name</string>
+  <string name="customdrugssearch_item_name_separator">, </string>
```

Comment:
Just wondering if it would be better for the localisation team to have two different string resources, `drug_with_dosage` and `drug_with_dosage_and_frequency`, so they can make a better decision when adding the translations.

Reply:
I initially went with that, but dosage and frequency is nullable. So sometimes there is a white space before the separator. That's why I switch to a string builder.